### DISCUSSION TITLE
Prompt for vmbr interface when multiple are detected.

### DISCUSSION THREAD

**Discussion by jordantrizz** [community-scripts/ProxmoxVE#1] [Ideas]
Internet access is required to complete software installations; if not fulfilled, the installation will fail. To my knowledge, there isn't another setting that would fail an installation. If there is more than one vmbr network interface, a prompt to select the appropriate vmbr should occur.

**Comment by tteck** [comment]
Have you looked through "Advanced" settings instead of utilizing "Default"?

![Screenshot 2024-11-02 1 43 57 AM](https://github.com/user-attachments/assets/b43d0518-c0e3-41b1-aa18-e38eda456b8b)

**Reply by jordantrizz** [reply]
Yes, this is how I have to do the installs each time for a single host due to its network configuration. The host is an OVH UK bare-metal machine that requires additional IPs to be source-routed to internal IPs for each Proxmox instance.

https://help.ovhcloud.com/csm/en-gb-dedicated-servers-proxmox-network-hg-scale?id=kb_article_view&sysparm_article=KB0043899

So I have to use vmbr1 instead; which is a separate private network with NAT using the Proxmox host IP. Otherwise, the installation will fail to proceed. Then I configure Cloudflare tunnel to access the container externally.

Changing the memory, CPU, storage, OS type, etc, will not cause the installation to fail. However, networking will, as it's a requirement that internet is available to proceed with an installation. When more than one network bridge is detected, prompt which bridge to utilize. 

When installing uptime-kuma, for example, it requires 8-10 different options to configure just to change the bridge. Versus two to three potentially.

You can totally disagree, I'm just giving feedback.

**Comment by jordantrizz** [comment]
I updated the discussion title and message to be more specific.
